.changes/unreleased/operator-Added-20250630-182854.yaml

```diff
@@ -0,0 +1,4 @@
+project: operator
+kind: Added
+body: New option to opt out from generating user password when ValueFrom is set, but the Kubernetes Secret does not exist.
```

Comment:
When possible, provide key names such that a (well informed) user wouldn't have to refer to another form of documentation to begin using a newly added feature. 

```suggestion
body: Added `noGenerate` to `Password`. When `true`, it prevents the operator from generating non-existent Secrets and non-existent Secret keys that are specified via `valueFrom`
```